lib/jobs/ansible-job.js

```diff
@@ -65,6 +86,24 @@ function ansibleJobFactory(
             return self._publishAnsibleResult(self.routingKey, data);
         })
         .then(function() {
+            try {
```

Comment:
You can leave out the try/catch blocks and handle it in the .catch();
return fs.readFileAsync(self.outFileName, 'utf8').then(function(contents) {  
  return fs.unlinkAsync(self.outFileName);
}).catch(function(err) {
  // error handling
});